host-controller/src/main/resources/org/jboss/as/host/controller/descriptions/LocalDescriptions.properties

```diff
@@ -71,12 +71,16 @@ host.server-config=Host-level configurations for the servers that can run on thi
 host.server=Servers currently running on the host
 host.socket-binding-group=The socket binding group currently configured for use on the host controller.
 host.shutdown=Shuts down the host and its servers
+host.shutdown.suspend-timeout=Suspend timeout in seconds used to suspend the servers. If this is zero (the default) then the server(s) will suspend immediately. A value larger than zero means the server(s) will wait up to this many seconds for all active requests to finish. A value smaller than zero means that the server(s) will wait indefinitely for all active requests to finish.
 host.shutdown.restart=If true, once shutdown the host controller will be restarted again
 host.state=The current state of the host controller; either STARTING, RUNNING or RESTART_REQUIRED
 host.master=Whether this host is master host for the domain; i.e. whether this process is acting as the Domain Controller.
 host.resolve-expression-on-domain=Operation that accepts an expression as input (or a string that can be parsed into an expression) and resolves it against the local system properties and environment variables on all servers managed by this host controller.
 host.resolve-expression-on-domain.expression=The expression to resolve.
 host.resolve-expression-on-domain.reply=The resolved expression, or the string form of the original input value if it did not represent an expression.
+host.suspend-servers=Suspends operations on all servers managed by this host controller. All current operations will be allowed to finish, and new operations will be rejected.
```

Comment:
Good point Jan. Indeed the incorrect description is in `host.suspend-servers.suspend-timeout` instead of `host.suspend-servers`, in general, all the descriptions need a review to make them a bit more consistent.

For suspend operations, the `suspend-timeout` is referring to the timeout of the suspend operation execution itself, the server does not wait for all active request to finish, it waits up X seconds before returning from the suspend operation.

For operations that trigger a shutdown/stop, the description is a bit different since as soon as the suspend operation returns, the server is shut down, which effectively finishes any inflight operations.

I've done a review of all of those description; @jstourac could you review them?

Reply:
@yersan thanks for this update. I've looked through and I think that messages are good now. Still, I have put three comments... but to be honest, I am not sure whether the proposed changes are really necessary - I'll leave it up to you whether it makes sense to make another update for these three messages.

Reply:
Your notes make sense to me, so I changed them, take a look now